packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -3145,8 +3143,7 @@ const ruleset = {
         },
         ValidateSegmentsInNestedResourceListOperation: {
```

Comment:
>ValidateSegmentsInNestedResourceListOperation [](http://example.com/codeflow?start=8&length=45)

Wasnt this missing the link to the md file? We should fix that before the upgrade

Reply:
updated !
will move it to next release

Reply:
lets wait for Tejaswi's fix to get in and then remove this from staging before upgrading